Tests/MoyaSugarTests/MoyaSugarProviderTests.swift

```diff
@@ -69,23 +70,9 @@ class MoyaSugarProviderTests: XCTestCase {
     let provider = MoyaSugarProvider<GitHubAPI>(endpointClosure: endpointClosure)
     let endpoint = provider.endpoint(GitHubAPI.userRepos(owner: "devxoul"))
     XCTAssertNotNil(endpoint.httpHeaderFields)
-    XCTAssertEqual(endpoint.httpHeaderFields!, ["Authorization": "MySecretToken"])
-  }
-
-  func testHTTPHeaderFields_extendedByEndpointClosure() {
-    let endpointClosure: (GitHubAPI) -> Endpoint<GitHubAPI> = { target in
-      let defaultEndpoint = MoyaProvider.defaultEndpointMapping(for: target)
-      return defaultEndpoint
-        .adding(newHTTPHeaderFields: target.httpHeaderFields ?? [:])
-        .adding(newHTTPHeaderFields: ["Authorization": "MySecretToken"])
-    }
-    let provider = MoyaSugarProvider<GitHubAPI>(endpointClosure: endpointClosure)
-    let endpoint = provider.endpoint(GitHubAPI.userRepos(owner: "devxoul"))
-    XCTAssertNotNil(endpoint.httpHeaderFields)
-    XCTAssertEqual(endpoint.httpHeaderFields!, [
-      "Authorization": "MySecretToken",
-      "Accept": "application/json",
-    ])
+    XCTAssertTrue(endpoint.httpHeaderFields!.contains(where: { (key, value) -> Bool in
```

Comment:
The behavior of `MoyaProvider.defaultEndpointMapping` has changed so we can just test whether the `httpHeaderFields` contains both values for keys: `MySecretToken`, `Authorization`. Maybe something like:

```swift
XCTAssertTrue(endpoint.httpHeaderFields?["Accept"], "application/json")
XCTAssertTrue(endpoint.httpHeaderFields?["Authorization"], "MySecretToken")
```

Reply:
It will change to 
```
    XCTAssertEqual(endpoint.httpHeaderFields!, [
      "Authorization": "MySecretToken",
      "Accept": "application/json",
    ])
```